src/main/resources/GoogleApps/Translations.xml

```diff
@@ -36,7 +36,128 @@
   <minorEdit>false</minorEdit>
   <syntaxId>plain/1.0</syntaxId>
   <hidden>true</hidden>
-  <content/>
+  <content>admin.googleapps=Google Apps
```

Comment:
I'm surprised that all these keys were added. Maybe the keys are kept on a separate file in the file system and added to the content at build time?